### DISCUSSION TITLE
Absorption voltage and a float voltage VS absorption time

### DISCUSSION THREAD

**Discussion by cyborgxxl** [Louisvdw/dbus-serialbattery#510] [Q&A] [answered]
I'm not sure if this topic has been discussed before...

All Victron charge controllers have an absorption voltage and a float voltage. There is also an absorption time for this.

If I now control the charge via my JK-BMS and the serialbattery driver, this set absorption time seems to be ignored. Can anybody confirm this?

If so, it would be great if the serialbattery driver would also consider the absorption time so the battery cells aren't kept at 3.55V all the time. In the long run, this is certainly not optimal for cell life.

**Comment by mr-manuel** [comment, marked as answer]
Hi, please check this issue https://github.com/Louisvdw/dbus-serialbattery/issues/421. Probably this should answer your question.

**Reply by cyborgxxl** [reply]
Thank you! I didn't see this... ;-)
